BrushUpCode/src/State.java

```diff
@@ -0,0 +1,8 @@
+public enum State {
+    NONE,
+    ACTION,
+    OFF,
+    MAKINGCOFFE,
+    FILLING,
```

Comment:
@maria-salmasian does this compile ?

Reply:
No, it it just for the state of the coffeeMachine it s an enum class

Reply:
I didn't know if the state was necessary or not, so I added it anyway

Reply:
@maria-salmasian jan you don't need last `,` instead of you need `;` because by `,` it seems it should continue not end

Reply:
changed the last , to ;